app/models/event.rb

```diff
@@ -58,9 +58,9 @@ class Event < ApplicationRecord
       end
       string :keywords, multiple: true
       string :fields, multiple: true
-      time :start
+      time :start, trie: true
```

Comment:
A trie index is recommended for range search (which we also do on created_at). This means this PR needs a sunspot:reindex as part of its deploy.